examples/web-ui/material/pages/home.smash

```diff
@@ -0,0 +1,17 @@
+* Given I am on home page
+    Navigate to home page
+        On home page
+
+* Navigate to home page
+    Navigate to 'https://material.io/'
+        On home page
+
+* On home page
+    setHomePage +? {
```

Comment:
Why not just do 
```
* On home page {
    // code here
}
```

Reply:
Also, how about this:
```
* On Desktop
    * On Homepage {
        // Set all desktop homepage related props
    }

* On Mobile
    * On Homepage {
        // Set all mobile homepage related props
    }
```

Reply:
ps I know about `Init +?` in `browser.smash`. That's bad code and I'll most likely have to fix it ;)

Reply:
...fixed!